index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@1,700&display=swap" rel="stylesheet">
+</head>
+<body>
+      <nav>
+          <p class="logo">My Logo</p>
+          <ul>
+            <li><a href="#">portfolio</a></li>
+            <li><a href="#">about</a></li>
+            <li><a href="#">contact</a></li>
+          </ul>
+          <div class="menu">
+            <img src="./images/Icon - Menu.png" alt="">
+            <a href="toogle-nav">Menu</a>
+          </div>
+      </nav>
+      <header>
+        <div class="topimg">
+            <div class="imgorder imgorder1">
+              <img class="backimg backimg1" src="./images/Ellipse 11.png" alt="">
+              <img class="backimg backimg2" src="./images/Rectangle 4.png" alt="">
+            </div>
+            <img src="" alt="">
+            <div class="imgorder imgorder2">
+              <img class="backimg backimg3" src="./images/Rectangle 7.png" alt="">
+              <img class="backimg backimg4" src="./images/Ellipse 5.png" alt="">
+            </div>
+            <img class="icon" src="./images/Ellipse 7.png" alt="">
+            <div class="imgorder imgorder3">
+              <img class="backimg backimg5" src="./images/Rectangle 4.png" alt="">
+              <img class="backimg backimg6" src="./images/Ellipse 3.png" alt="">
+            </div>
+        </div>
+        <h1>I’m kirubel Glad <br/> to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p>LET’S CONNECT</p>
+        <img class="social"src="./images/social media icons.png" alt="">
```

Comment:
- [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag), Sorry this wasn't addressed by the previous reviewer but it's really part of the semantics, and HTML best practices that you should follow.

- Also those icons are meant to be links that lead the user to navigate to your social accounts, **and not an image.**

- [ ] After fixing the HTML structure of the social icons please consider fixing their layout and size as well.

### Your Design 
![image](https://user-images.githubusercontent.com/68030297/219722821-1cbeb207-160c-4359-b3f0-4847cd614636.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/219722476-93db25a6-7163-49aa-a879-90d23f162ef5.png)